src/include/gurt/types.h

```diff
@@ -97,6 +97,20 @@ typedef struct {
 	d_iov_t		*sg_iovs;
 } d_sg_list_t;
 
+/**
+ * c string buffer
+ */
+struct d_string_buffer_t {
+	/** c string status */
+	unsigned int status;
```

Comment:
status is unsigned int but is assigned negative values in code

Reply:
Done